backend/file_io.py

```diff
@@ -39,7 +39,7 @@ def log(*msg):
         tstamp = datetime.datetime.now()
         try:
             logfile = None
-            logfile = open('logfile.txt',"a")
+            logfile = open('otone_data/logfile.txt',"a")
```

Comment:
rolling in the logging module in the following branch:

https://github.com/OpenTrons/otone_backend/tree/201606270123100-desktop-logging